pages/settings/index.vue

```diff
@@ -0,0 +1,333 @@
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('operatingSystem') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.OperatingSystemDisplayName }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('architecture') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.SystemArchitecture }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="py-0">
+                    <span>{{ $t('vueClientVersion') }}</span>
+                  </v-col>
+                  <v-col class="py-0">
+                    <span>{{ vueVersion }}</span>
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col cols="3" class="d-flex justify-end">
+                <v-avatar tile size="96" color="grey">
+                  <v-icon>mdi-server</v-icon>
+                </v-avatar>
+              </v-col>
+            </v-row>
+          </v-card-text>
+        </v-card>
+        <v-card v-if="!$vuetify.breakpoint.mobile">
+          <v-card-text>
+            <v-row>
+              <v-col>
+                <p class="mb-0">
+                  <span
+                    v-html="
+                      sanitizeHtml(
+                        $t('links.poweredByJellyfinLink', {
+                          link: linkItems[0].link
+                        })
+                      )
+                    "
+                  />
+                  <br />
+                  <span
+                    v-html="
+                      sanitizeHtml(
+                        $t('links.readTheDocumentationLink', {
+                          link: linkItems[1].link
+                        })
+                      )
+                    "
+                  />
+                  <br />
+                  <span
+                    v-html="
+                      sanitizeHtml(
+                        $t('links.helpTranslateLink', {
+                          link: linkItems[1].link
+                        })
+                      )
+                    "
+                  />
+                  <br />
+                  <span
+                    v-html="
+                      sanitizeHtml(
+                        $t('links.reportAnIssueLink', {
+                          link: linkItems[2].link
+                        })
+                      )
+                    "
+                  />
+                </p>
+              </v-col>
+              <v-col cols="3" class="d-flex justify-end">
+                <v-img src="/icon.png" width="100%" :alt="$t('jellyfinLogo')" />
```

Comment:
```suggestion
                <v-img
                  contain
                  src="/icon.png"
                  width="100%"
                  :alt="$t('jellyfinLogo')"
                />
```
Sometimes logo is cut off. Containing it should fix it